src/Chainweb/Version.hs

```diff
@@ -1033,6 +1034,13 @@ chainweb218Pact Development = (>= 500)
 chainweb218Pact (FastTimedCPM g) | g == petersonChainGraph = (> 60)
 chainweb218Pact _ = (> 24)
 
+chainweb219Pact :: ChainwebVersion -> BlockHeight -> Bool
+chainweb219Pact Mainnet01 = error "fill" -- Todo: comment times
+chainweb219Pact Testnet04 = error "fill" -- Todo: comment times
+chainweb219Pact Development = (>= 550)
+chainweb219Pact (FastTimedCPM g) | g == petersonChainGraph = (> 70)
```

Comment:
How do we determine this value?

Reply:
Usually just a couple of blocks after the latest petersen chain fork. Last fork was block 60, this could be 65 or 70. Same with devnet.